modules/stochastic_tools/src/base/StochasticToolsApp.C

```diff
@@ -6,14 +6,22 @@
 #include "StateSimTester.h"
 #include "StateSimRunner.h"
 
-// distributions
+// Distributions
 #include "UniformDistribution.h"
-// samplers
+#ifdef LIBMESH_HAVE_BOOST
```

Comment:
I agree, the reason I did it this way was because of the templated BoostDistribution, but I can probably figure out a way to do this and have the general template base. I will work on this.